.circleci/config.yml

```diff
@@ -712,3 +723,40 @@ jobs:
             export IMAGE_TAG=$(bash ~/project/scripts/pipeline/set_environment_variables/set_image_tag.sh $CIRCLE_BRANCH $CIRCLE_SHA1) >> $BASH_ENV
                 echo $IMAGE_TAG
             python scripts/pipeline/check_ecr_scan_results/aws_ecr_scan_results.py --search online-lpa --tag ${IMAGE_TAG}
+
+  workspace_cleanup:
```

Comment:
Are these the workspace cleanup changes from the other ticket?

Reply:
my mess up checked out over the wrong branch 👎  - fixing now